<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wdgStrategyResults</class>
 <widget class="QWidget" name="wdgStrategyResults">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1048</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="lblTitulo">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Strategy results</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="lblPixmap">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
         <property name="pixmap">
          <pixmap resource="../images/xulpymoney.qrc">:/xulpymoney/tools-wizard.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Select your strategy results</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmbStrategies"/>
       </item>
       <item>
        <widget class="QToolButton" name="cmdStrategyAdd">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../images/xulpymoney.qrc">
           <normaloff>:/xulpymoney/add.png</normaloff>:/xulpymoney/add.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="cmdStrategyEdit">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../images/xulpymoney.qrc">
           <normaloff>:/xulpymoney/editar.png</normaloff>:/xulpymoney/editar.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="cmdStrategyDelete">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../images/xulpymoney.qrc">
           <normaloff>:/xulpymoney/list-remove.png</normaloff>:/xulpymoney/list-remove.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>From</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="wdgDatetime" name="wdgDtFrom" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>To</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="wdgDatetime" name="wdgDtTo" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="cmbManagerSelector" name="cmsInvestments" native="true"/>
     </item>
     <item>
      <widget class="QTabWidget" name="tab_4">
       <property name="currentIndex">
        <number>2</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Current operations</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QWidget" name="mqtwCurrent" native="true"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Historical Operations</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QWidget" name="mqtwHistorical" native="true"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>Dividends</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QWidget" name="mqtwDividends" native="true"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>wdgDatetime</class>
   <extends>QWidget</extends>
   <header location="global">wdgDatetime</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>cmbManagerSelector</class>
   <extends>QWidget</extends>
   <header location="global">frmSelector</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../images/xulpymoney.qrc"/>
  <include location="../images/xulpymoney.qrc"/>
 </resources>
 <connections/>
</ui>
